src/lib/assets/icons/download_list/easy.svg

```diff
@@ -0,0 +1,33 @@
+<svg width="136" height="74" viewBox="0 0 136 74" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Icons that describe how difficult the action is.  Worked with Emily and Stockard to categorize the different scenarios.  See https://sagebionetworks.jira.com/browse/PORTALS-1947 for more info.